### PR TITLE
Adapt README for tlbc setup instead of laika

### PR DESCRIPTION
**Pull request by weilbith**
Closes: #593

### CODE REVIEW

**Comment by cducrest** [comment]
Line 26, it is the wrong url, it should be https://netstats.tlbc.trustlines.foundation/

**Comment by cducrest** [comment]
Line 26, it is the wrong url, it should be https://netstats.tlbc.trustlines.foundation/

File: README.md [hunk 1]
```diff
@@ -30,9 +31,17 @@ possible to avoid Docker altogether and run everything directly on the host mach
 
 Before starting the installation process, please have a look at the system requirements and the note on security.
```
Comment (cducrest) [inline README.md]: Before starting the installation process, please have a look at the following sections on the distinction between Laika and the Trustlines Blockchain, system requirements, and security.

File: README.md [hunk 2]
```diff
@@ -30,9 +31,17 @@ possible to avoid Docker altogether and run everything directly on the host mach
...
 
+### Networks
+
+There are two blockchain networks related to this project. The test network is
+called Laika. The productive network is called the Trustlines Blockchain. Often
+this gets abbreviated to `tlbc`, especially for technical components. The
+instructions within this documentation primarily focus on the Trustlines
+Blockchain.
```
Comment (cducrest) [inline README.md]: I would rename the section to "TLBC and Laika" or "The Trustlines Blockchain and Laika" or something like that and not talk about networks.

Comment (cducrest) [inline README.md]: I would write:
"There are two different blockchains related to this project. The first one is a testnet called Laika. The second one is considered to be the main net and called the Trustlines Blockchain or TLBC. The instructions within this document primarily focus on the Trustlines Blockchain.

File: README.md [hunk 3]
```diff
@@ -115,50 +129,50 @@ $ docker run trustlines/tlbc-testnet --help
...
-    -v $(pwd)/trustlines/config:/config/custom \
-    -v $(pwd)/trustlines/enode:/config/network \
-    -v $(pwd)/trustlines/shared:/shared/ \
+$ mkdir -p tlbc/databases/tlbc tlbc/config tlbc/enode tlbc/shared
+$ docker run -d --name tlbc-node --network network-tlbc \
+    -v $(pwd)/tlbc/databases/tlbc:/data \
+    -v $(pwd)/tlbc/config:/config/custom \
+    -v $(pwd)/tlbc/enode:/config/network \
+    -v $(pwd)/tlbc/shared:/shared/ \
     -p 30300:30300 -p 30300:30300/udp \
```
Comment (cducrest) [inline README.md]: We now use port 30302: https://github.com/trustlines-protocol/blockchain/blob/master/tools/quickstart/quickstart/configs/tlbc/docker-compose.yaml#L19
I guess better be consistent

File: README.md [hunk 4]
```diff
@@ -115,50 +129,50 @@ $ docker run trustlines/tlbc-testnet --help
...
-    -v $(pwd)/trustlines/shared:/shared/ \
+$ mkdir -p tlbc/databases/tlbc tlbc/config/keys/tlbc tlbc/enode tlbc/shared
+$ cp /path/to/your/keystore/file.json tlbc/config/keys/tlbc/account.json
+$ echo "<passphrase_for_keystore_file>" > tlbc/config/pass.pwd
+$ docker run -d --name tlbc-node --network network-tlbc \
+    -v $(pwd)/tlbc/databases/tlbc:/data \
+    -v $(pwd)/tlbc/config:/config/custom \
+    -v $(pwd)/tlbc/enode:/config/network \
+    -v $(pwd)/tlbc/shared:/shared/ \
     -p 30300:30300 -p 30300:30300/udp \
```
Comment (cducrest) [inline README.md]: Change port here if you changed above.

File: README.md [hunk 5]
```diff
@@ -259,8 +272,10 @@ We refer to the documentation of the individual components:
 - [Monitor](https://github.com/trustlines-protocol/tlbc-monitor)
 - [Bridge](https://github.com/trustlines-protocol/blockchain/tree/master/tools/bridge)
 
-For the Laika node, make sure it uses the correct [chain spec file](), that the right TCP and UDP ports are used
-(30300), and that the JSON RPC APIs `web3`, `eth`, and `net` are enabled.
+For the Trustlines Blockchain node, make sure it uses the correct chain
+specification file (`./chain/tlbc/tlbc-spec.json`), that the right TCP and UDP
+ports are used (30300), and that the JSON RPC APIs `web3`, `eth`, and `net` are
```
Comment (cducrest) [inline README.md]: ports

File: README.md (hunk 1 above)
Comment (weilbith) [reply README.md]: :heavy_check_mark:

File: README.md (hunk 2 above)
Comment (weilbith) [reply README.md]: :heavy_check_mark:

Comment (weilbith) [reply README.md]: :heavy_check_mark:

File: README.md (hunk 3 above)
Comment (weilbith) [reply README.md]: :heavy_check_mark:

File: README.md (hunk 4 above)
Comment (weilbith) [reply README.md]: :heavy_check_mark:

File: README.md (hunk 5 above)
Comment (weilbith) [reply README.md]: :heavy_check_mark:

**Pull request merged**